Documentation/install/guides/kops.rst

```diff
@@ -4,37 +4,167 @@
+        aws iam create-user --user-name kops
+        aws iam add-user-to-group --user-name kops --group-name kops
+        aws iam create-access-key --user-name kops
+
+
+kops requires a creation of a dedicated s3 bucket in order to store the state and representation of the cluster. You will need to change the bucket name and provide your unique bucket name (for example a reverse of FQDN added with short description of the cluster). Also make sure to use the region where you will be deploying the cluster. 
+
+.. code:: bash
+
+        aws s3api create-bucket --bucket prefix-example-com-state-store --region us-west-2 --create-bucket-configuration LocationConstraint=us-west-2
+        export KOPS_STATE_STORE=s3://prefix-example-com-state-store
+
+Above steps are sufficient for getting a working cluster installed. Please consult `kops aws documentation <https://github.com/kubernetes/kops/blob/master/docs/aws.md>`_ for more detailed setup
+
+
+Cilium Prerequisites
+====================
+
+* Cilium requires minimum kernel version to be 4.8
+* Cilium requires minimum etcd version as 3.1  
+* Kubernetes with CRD validation (Recommended) (more on this after initial dry run setup of the cluster)
+
+In this guide we will use the etcd version as 3.1.11 and the latest CoreOS stable image  which satisfies the minimum kernel version requirement of cilium. To get the latest CoreOS ami image, you can change the region value to your choice in the command below.
+
+.. code:: bash
+        
+        aws ec2 describe-images --region=us-west-2 --owner=595879546273 --filters "Name=virtualization-type,Values=hvm" "Name=name,Values=CoreOS-stable*" --query 'sort_by(Images,&CreationDate)[-1].{id:ImageLocation}'
+
+.. code:: json
+
+        {
+                "id": "595879546273/CoreOS-stable-1745.5.0-hvm"
+        }
+
+
+Creating the Cluster
+====================
+
+* Note that you will need to specify the ``--master-zones`` and ``--zones`` for creating the master and worker nodes. The number of master zones should be odd (1, 3, ...) for the HA. For simplicity, you can just use 1 region.
+* the cluster ``NAME`` variable should end in ``k8s.local`` to use the gossip protocol. 
+
+.. code:: bash
+
+        export NAME=cilium.k8s.local
+        export KOPS_FEATURE_FLAGS=SpecOverrideFlag 
+        kops create cluster  --state=${KOPS_STATE_STORE}  --node-count 3 --node-size t2.medium --master-size t2.medium --topology private --master-zones us-west-2a,us-west-2b,us-west-2c --zones us-west-2a,us-west-2b,us-west-2c --image 595879546273/CoreOS-stable-1745.5.0-hvm --networking cilium --override "cluster.spec.etcdClusters[*].version=3.1.11" --kubernetes-version 1.10.3  --cloud-labels "Team=Dev,Owner=Admin"  ${NAME}
+
+
+You may be prompted to create a ssh public-private key pair.
+
+.. code:: bash
+
+        ssh-keygen
+
+
+(Please see appendix for details on the flags used to create the cluster.)
```

Comment:
Is it possible to link to the appendix here? Would be nice to make it clickable.